app/markdownhighlighter.cpp

```diff
@@ -22,6 +22,7 @@
 #include <QTextLayout>
```

Comment:
Don't hesitate to add your copyright, if you make such substantial changes to a file.